### ISSUE TITLE
convenient to provide Inception-ResNet-v2, Inception V4 effects?

### ISSUE THREAD

**Issue by Shuolongbj** [aaron-xichen/pytorch-playground#2]
Pretty work, can be further provided Top-5 Accuracy?

FYI.
TF-slim:
https://github.com/tensorflow/models/tree/master/slim

**Comment by aaron-xichen** [comment]
I am not sure whether we could get these two trained models directly in pytorch. Otherwise we may need to train them for ourselves. Top-5 Accuracy will be provided further, maybe couple of days I think.

**Comment by Shuolongbj** [comment]
Thanks, look forward to further big progress
